tmail-backend/tmail-third-party/openpaas/src/test/resources/docker-openpaas-setup.yml

```diff
@@ -1,4 +1,4 @@
-version: "2"
+version: "2.7"
```

Comment:
Is it fine to upgrade to `version: '3'` ? (similar to other docker-compose sample files)